test/network_autoconfiguration_test.rb

```diff
@@ -47,6 +48,14 @@ def probe_netcard_factory(num)
 end
 
 describe Yast::NetworkAutoconfiguration do
+  let(:yast_config) { Y2Network::Config.new(source: :sysconfig) }
+  let(:system_config) { yast_config.copy }
+
+  before do
+    Yast::Lan.add_config(:yast, yast_config)
+    Yast::Lan.add_config(:system, system_config)
```

Comment:
I'd prefer ```Y2Network::Config.add``` here. You don't need dependency on ```Lan``` module then. And ```add_config``` is only wrapper for it anyway.